app/views/users/new.html.erb

```diff
@@ -0,0 +1,47 @@
+
+<%# <form action="/users" method="post" accept-charset="UTF-8">
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+  <%# <label for="username">Username:</label>
+  <input type="text" id="username" name="username"><br><br>
```

Comment:
- [ ] The names of each input should follow the pattern `user[<field_name>]`.
- [ ] Make sure this commented out form is actually working.
You should have followed the work flow Odin project gave you, 
first make sure the form using `<form>` tag is working.
then comment it out
then make the form_tag form
make sure it's working
and the comment it out.